packages/components/src/filters/advanced/search-filter.js

```diff
@@ -40,7 +40,12 @@ class SearchFilter extends Component {
 	}
 
 	updateLabels( selected ) {
```

Comment:
@Aljullu Does this need to be called still from `componentDidUpdate()`?  It looks like maybe searching in the table is supposed to open the advanced filter as well, but I'm not seeing this happening on master.

Commit for reference - https://github.com/woocommerce/wc-admin/pull/1085/commits/6247cc30572980afb04f1686e16300a6e85898ed

Reply:
Good catch! At that time, the table search input and the _Advanced Filters_ were connected, but that's no longer the case, so I think it can be safely removed.

I created #1544 to address that.